app/src/main/res/values-de/strings.xml

```diff
@@ -75,31 +75,31 @@
     <string name="action_start">Start</string>
     <string name="action_stop">Stop</string>
     <string name="automatic_refresh">Täglich aktualisieren</string>
-    <string name="automatic_refresh_description">Tägliche Aktualisierungen finden statt, wenn das Gerät lädt, in Ruhe ist, und die Verbindung gebührenfrei ist.</string>
+    <string name="automatic_refresh_description">Tägliche Aktualisierungen, wenn das Gerät geladen wird, im Ruhezustand ist und die Verbindung gebührenfrei ist.</string>
     <string name="watchdog">Überwache Verbindung</string>
-    <string name="watchdog_description">Prüfe die Verbindung in ansteigenden Abständen und verbinde erneut, falls sie nicht mehr funktioniert.</string>
-    <string name="legend_host_intro">Einträge werden der Reihenfolge nach bearbeitet. Enthalten mehrere den gleichen Host, so wird der letzte Angewandt. Die Aktionen sind:</string>
+    <string name="watchdog_description">Prüfe die Verbindung in regelmäßigen Abständen und verbinde erneut, falls diese nicht mehr funktioniert.</string>
+    <string name="legend_host_intro">Einträge werden der Reihenfolge nach bearbeitet. Enthalten mehrere den gleichen HOST, so wird der letzte angewandt. Die Aktionen sind:</string>
     <string name="legend_host_ignore">Ignoriere diesen Eintrag</string>
-    <string name="legend_host_deny">Verweigere Zugriff auf Hosts in diesem Eintrag</string>
-    <string name="legend_host_allow">Erlaube Zugriff auf Hosts in diesem Eintrag</string>
+    <string name="legend_host_deny">Verweigere Zugriff auf HOSTS in diesem Eintrag</string>
+    <string name="legend_host_allow">Erlaube Zugriff auf HOSTSS in diesem Eintrag</string>
     <string name="use_dns_server">An</string>
     <string name="do_not_use_dns_server">Aus</string>
-    <string name="switch_onboot_description">Starte DNS66 beim Einschalten des Gerätes, falls es beim Ausschalten aktiv war.</string>
+    <string name="switch_onboot_description">Starte DNS66 beim Einschalten des Gerätes, falls es beim ausschalten aktiv war.</string>
     <string name="activity_edit_dns_server">DNS Server bearbeiten</string>
     <string name="activity_edit_filter">Filter bearbeiten</string>
-    <string name="notification_paused_text">Zum Fortsetzen tippen.</string>
+    <string name="notification_paused_text">Zum Fortsetzen antippen.</string>
     <string name="notification_paused_title">DNS66 unterbrochen</string>
     <string name="notification_action_pause">Unterbrechen</string>
     <string name="whitelist_on_vpn">Keine Apps</string>
     <string name="whitelist_not_on_vpn">Alle Apps</string>
     <string name="whitelist_intelligent">System Apps (außer Browser)</string>
     <string-array name="whitelist_defaults">
-        <item>Keine Apps umgehen standardmäßig</item>
-        <item>Alle Apps umgehen standardmäßig</item>
-        <item>System Apps umgehen standardmäßig</item>
+        <item>Keine Apps umgehen (Standard)</item>
```

Comment:
That's somewhat confusing, as it can be read as meaning that all choices are the default of the app.